### PR TITLE
Updates for recipes

### PR DESCRIPTION
**Pull request by aarontp**
A few updates for recipes:
- Adds three new recipes (triage, triage-windows and a kitchen sink recipe for 'everything') and deletes the test one
- Update server to use the 'globals'
- Removed the `*_file` globals since those should be serialzed into the request at the client (otherwise the server/worker won't have access to the files unless they happened to be on shared storage).
-  Separated `task_config` into a different default class variable `TASK_CONFIG` because `task_config` was being referenced as an object attribute and class variable.
- Add new `--recipe_path` flag to be able to set full path of recipe in addition to the existing recipe name
- Try to append '.yaml' to recipe if we can't find it first
- Moves some more code into `get_task_recipe()`
- Add a few log messages and update docstrings
-

### CODE REVIEW

**Review by wajihyassine — CHANGES_REQUESTED** [review]
LGTM other than the nitpick with changing the filename of everything.yaml!

File: turbinia/config/recipes/everything.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+globals:
```
[the remark this answers is not in the document]
Comment (aarontp) [reply turbinia/config/recipes/everything.yaml:1]: After our internal discussion, how does just `all` sound?

[the remark this answers is not in the document]
Comment (wajihyassine) [reply turbinia/config/recipes/everything.yaml:1]: `all` SGTM/users should be able to understand. Can also make mention to it in documentation if anything

[the remark this answers is not in the document]
Comment (aarontp) [reply turbinia/config/recipes/everything.yaml]: Ok.  PTAL.  I renamed the recipe and added a comment at the top.

**Review by wajihyassine — APPROVED** [review]
LGTM!

**Pull request merged**